src/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/_container_service_client.py

```diff
@@ -121,6 +121,7 @@ def models(cls, api_version=DEFAULT_API_VERSION):
            * 2021-07-01: :mod:`v2021_07_01.models<azure.mgmt.containerservice.v2021_07_01.models>`
            * 2021-08-01: :mod:`v2021_08_01.models<azure.mgmt.containerservice.v2021_08_01.models>`
            * 2021-09-01: :mod:`v2021_09_01.models<azure.mgmt.containerservice.v2021_09_01.models>`
+           * 2021-10-01: :mod:`v2021_10_01.models<azure.mgmt.containerservice.v2021_10_01.models>`
```

Comment:
Please modify the 59th line and change it to `DEFAULT_API_VERSION = '2021-10-01'`. You can copy this file directly from the SDK.